src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -164,14 +164,10 @@ private void switchTimelines(final Timeline activeTimeline, final Timeline nextT
                 Thread.currentThread().interrupt();
                 throw new TimelineException("Failed to switch timeline for:" + activeTimeline.getEventType());
             } catch (final Exception ex) {
+                LOG.error(ex.getMessage(), ex);
```

Comment:
Yes, you are right, but this is kind of exception case here: If exception is thrown here and after that another exception is thrown in finally block then we will lose this exception stack trace.